docs/api/commands/visit.mdx

```diff
@@ -354,9 +354,10 @@ cy.visit('index.html').then((contentWindow) => {
 
 ### User agent
 
-Trying to change the `User-Agent`? You can set the `userAgent` as a
-[configuration value](/guides/references/configuration#Browser) in your
-configuration file.
+Trying to change the `User-Agent`?
+
+See
+[how to override the default user agent the browser uses](../../faq/questions/using-cypress-faq#Can-I-override-or-change-the-default-user-agent-the-browser-uses).
```

Comment:
the "../.." step to create the relative path isn't necessary in the Docs site. You can use:

```suggestion
[how to override the default user agent the browser uses](/faq/questions/using-cypress-faq#Can-I-override-or-change-the-default-user-agent-the-browser-uses).
```